webapp/mock.py

```diff
@@ -0,0 +1,6 @@
+#mock data to test
```

Comment:
переделай в докстринг
```suggestion
"""This module contains mock data to test.""""
```